src/xmp-parser.js

```diff
@@ -8,17 +8,57 @@ export default {
     read
 };
 
+const tagTypes = {
+
+    // DJI
+    'drone-dji:CamReverse': 'bool',
+    'drone-dji:GimbalReverse': 'bool',
+    'drone-dji:RtkFlag': 'bool',
+
+    // CRS
+    'crs:Version': 'string',
+
+    // Camera
+    'Camera:IsNormalized': 'bool',
+};
+
 function read(xmpString) {
     try {
         const doc = getDocument(xmpString);
         const rdf = getRDF(doc);
 
-        return parseXMPObject(convertToObject(rdf, true));
+        const xmp = parseXMPObject(convertToObject(rdf, true));
+        Object.keys(xmp).forEach((key) => {
+            xmp[key] = parseValue(key, xmp[key]);
+        });
+        return xmp;
     } catch (error) {
         return {};
     }
 }
 
+function parseValue(key, value) {
+    if (value instanceof Array) {
+        return value.map((val) => parseValue(key, val));
+    } else if (value instanceof Object && 'value' in value) {
+        value.value = parseValue(key, value.value);
+        return value;
+    } else if (tagTypes[key]) {
+        if (tagTypes[key] === 'bool') {
+            return ['True', 'true', '1'].includes(value);
+        } else if (tagTypes[key] === 'string') {
+            return value;
+        }
+    } else if (!isNaN(value) && value !== '') {
+        return +value;
```

Comment:
Minor: I like Number() better than implicit casting like this

Reply:
Fixed

Reply:
Also did the spreading above as suggested